### PR TITLE
Rest of Support Services finished, Redundant code removed

### PR DESCRIPTION
**Pull request by rohands**
Many layouts had been using the same xml structure which consists of one title and a textbox. Removed all instances of it and replaced it with one main layout which inflates all the other ones. ( No redundancy)

Apart from this, added Mythbusters, Confidentiality and After Assault

screenshot_20160719-035757

screenshot_20160718-213255

### CODE REVIEW

File: app/src/main/java/com/peacecorps/pcsa/support_services/StepsFragment.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.peacecorps.pcsa.R;
+
+/**
+ * Steps in Reporting
+ *
+ * @author Buddhiprabha Erabadda
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/support_services/StepsFragment.java]: Is it a copied comment?

**Comment by sandarumk** [comment]
PR looks okay. Please resolve the comment issue. I think since the class is added new the comment block should change.

File: app/src/main/java/com/peacecorps/pcsa/support_services/StepsFragment.java (hunk 1 above)
Comment (Buddhiprabha) [inline app/src/main/java/com/peacecorps/pcsa/support_services/StepsFragment.java]: I think this is a class name change. If so what @rohands has done is correct.

Comment (rohands) [inline app/src/main/java/com/peacecorps/pcsa/support_services/StepsFragment.java]: Yes, @Buddhiprabha is right
This layout had been previously created by Buddhiprabha. I have only modified it a bit in the XML to suit the new mockups. Also, in the java file, I have made only the required changes to make it a Fragment. 

Hence, I copied the comment of Steps class. 
Should I make changes to this?

Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/support_services/StepsFragment.java]: No then. Keep it as it is. I will merge the PR

**Pull request merged**